src/media/inner.rs

```diff
@@ -1322,10 +1321,11 @@ impl MediaInner {
         self.simulcast = Some(s);
     }
 
-    pub fn ssrc_rx_for_rid(&self, repairs: Rid) -> Option<Ssrc> {
+    pub fn ssrc_rx_for_rid(&self, repairs: Option<Rid>, ignore_ssrc: Ssrc) -> Option<Ssrc> {
```

Comment:
Perhaps name this find_primary_ssrc(&self, rid: Option<Rid>, rtx_ssrc: Ssrc).  That would be more clear.